1906-12-03

```diff
@@ -0,0 +1,2109 @@
+                            <cell>is due on or about</cell>
+                            <cell><date when="1905-07-16">July 16</date></cell>
+                        </row>
+                        <row>
+                            <cell>Ellerman <name>S.S. City of Dundee</name></cell>
+                            <cell>expected from</cell>
+                            <cell><placeName>Glasgow</placeName>, <placeName>Gibraltar</placeName> &amp; <placeName>Malta</placeName></cell>
+                            <cell>is due on or about</cell>
+                            <cell><date when="1905-07-25">July 25</date></cell>
+                        </row>
+                        <row>
+                            <cell>Westcott <name>S.S. Plymothian</name></cell>
+                            <cell>expected from</cell>
+                            <cell><placeName>Antwerp</placeName>, <placeName>London</placeName> &amp; <placeName>Malta</placeName></cell>
+                            <cell>is due on or about</cell>
+                            <cell><date when="1905-07-25">July 25</date></cell>
+                        </row>
+                        <row>
+                            <cell>Ellerman <name>S.S. City of Oxford</name></cell>
+                            <cell>expected from</cell>
+                            <cell><placeName>Liverpool</placeName> &amp; <placeName>Malta</placeName></cell>
+                            <cell>is due on or about</cell>
+                            <cell><date when="1905-07-30">July 30</date></cell>
+                        </row>
+                    </table>
+                    <p>Ellerman <name>S.S. Britannia</name> now on the berth for <placeName>Liverpool</placeName> is expected to sail about
+                        the <date when="1905-07-25">25th inst.</date></p>
+                    <p>N. E. TAMVACO Alexandria agents <measure type="indexNo">23186-20-3-3</measure></p>
+                </div>
+                <div type="advert" xml:id="deg-ad-nkh01">
+                    <head>NEW KHEDIVIAL HOTEL, ALEXANDRIA.</head>
+                    <p>First-class Hotel. Situated in Rosetta Avenue, the finest quarter in the Town.
+                        Two mintes from Railway Station. Close to Conservatory and the Opera House.
+                        Lift. Electric Light Throughout. Perfect Sanitary Arrangements. Magnificent
+                        Ball, Reception, Reading, and Music Rooms. Bar and Smoking Room.</p>
+                    <p> HENRI CHAMOULLEAU, Proprietor.</p>
+                    <p>45</p>
+                    <p>FINE TERRACE ON THE AVENUE. - SPLENDID GARDEN. - OMNIBUS MEET ALL TRAINS AND
+                        STEAMERS. <measure type="indexNo">28-26</measure></p>
+                </div>
+                <div type="advert" xml:id="deg-ad-ngi01" xml:lang="fr">
+                    <head>Navigation Générale Italienne.</head>
+                    <p>Societes Reuni
```

Comment:
Please end the divisions of British India S. N. Company, Limited., and Hamburg & Anglo-American Nile Co. right after their body rather than ending them here. Every page should start with the division for page number<div type="page" n="*"/>, and you already include the page information inside specific advertisement divisions by end them in the end of the file.